src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -18,11 +21,16 @@ public PathService(StationService stationService, LineService lineService) {
         this.lineService = lineService;
     }
 
-    public PathResponse findShortestRoute(Long sourceStationId, Long targetStationId) {
+    public PathResponse findShortestRoute(Long sourceStationId, Long targetStationId, LoginMember loginMember) {
         Station sourceStation = stationService.findStationById(sourceStationId);
         Station targetStation = stationService.findStationById(targetStationId);
         PathFinder pathFinder = new PathFinder(sourceStation, targetStation, lineService.findLines());
-        return PathResponse.of(pathFinder.getShortestPathStationList(), pathFinder.getShortestPathDistance());
+
+        if (UserType.MEMBER.equals(loginMember.getUserType())) {
```

Comment:
enum 의 비교에서는 equals, == 비교 둘다 사용이 가능하지만 각각의 장단점이 있습니다. 찾아보시면 좋을 것 같구요 :)
개인적으론 enum 의 비교에서는 `앞에 상수 enum 을 위치하도록한 == 비교`를 선호합니다.
NPE 도 방지되면서 직관적이라고 생각해서입니다, 의견이 다른 분들도 많으니 팀 컨벤션을 따르는게 우선이지만 한번 생각해보셔도 좋을 것 같아요 ㅎㅎ

```suggestion
        if (UserType.MEMBER == loginMember.getUserType()) {
```

Reply:
enum비교는 == 로도 비교가 가능하군요..!
말씀처럼 NPE방지도 되고 직관적이며, 가독성도 더 좋은 것 같습니다 :)
차이점에 대해 따로 더 공부해보겠습니다..!